jre_emul/Makefile

```diff
@@ -207,8 +207,8 @@ $(BUILD_DIR)/.jre_dist_includes: $(PUBLIC_TRANSLATED_HEADERS:%=$(GEN_OBJC_DIR)/%
     $(PUBLIC_EMULATION_HEADERS:%=$(EMULATION_CLASS_DIR)/%) | $(DIST_INCLUDE_DIR)
 	@echo copying jre_emul header files
 	@(cd $(EMULATION_CLASS_DIR) && tar cf - \
-	    $(PUBLIC_EMULATION_HEADERS:$(EMULATION_CLASS_DIR)/%=%)) \
-	    | (cd $(DIST_INCLUDE_DIR); tar xfp -)
+	    $(PUBLIC_EMULATION_HEADERS:$(EMULATION_CLASS_DIR)/%=%) \
+	    | (cd ../$(DIST_INCLUDE_DIR); tar xfp -))
```

Comment:
Could you try using $(realpath $(DIST_INCLUDE_DIR)) instead of the ".."? That way we make no assumption about the depth of EMULATION_CLASS_DIR.